examples/point_cloud/main.cpp

```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
-#include "docopt.h"
+#include <command_parser.h>
+#include <aditof/log.h>
```

Comment:
can you link it to glog as well? we use this header only when glog dependency is set to off.

Reply:
Solved